.github/workflows/check-change.yml

```diff
@@ -1,8 +1,36 @@
-name: Check change
+name: Check changes/package versions
```

Comment:
For the future, we should decide on a generic name for this, as well as the filename.

Reply:
ok I came up with something, lemme know if this is good enough